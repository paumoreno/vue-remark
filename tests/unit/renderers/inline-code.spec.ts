import VueRemarkInlineCode from "@/renderers/inline-code.vue";
import { mount } from "@vue/test-utils";

describe(VueRemarkInlineCode.name, () => {
  test("snapshot", () => {
    const wrapper = mount(VueRemarkInlineCode, {
      propsData: {
        value: "import foo from 'bar';",
      },
    });

    expect(wrapper).toMatchSnapshot();
  });
});
