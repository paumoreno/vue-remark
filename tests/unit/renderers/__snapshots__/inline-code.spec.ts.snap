// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VueRemarkInlineCode snapshot 1`] = `<code>import foo from 'bar';</code>`;
